zerver/management/commands/generate_invite_links.py

```diff
@@ -53,4 +53,4 @@ def handle(self, *args, **options):
 
             prereg_user = PreregistrationUser(email=email, realm=realm)
             prereg_user.save()
-            print(email + ": " + Confirmation.objects.get_link_for_object(prereg_user, realm.host))
+            print(email + ": " + create_confirmation_link(prereg_user, realm.host, Confirmation.InVITATION))
```

Comment:
this is spelled wrong; I'm kinda surprised this didn't throw some sort of error.